Elmah.Io.ElasticSearch/ElasticClientSingleton.cs

```diff
@@ -18,9 +18,13 @@ public class ElasticClientSingleton : IDisposable
         private static ElasticClientSingleton _instance;
         public IElasticClient Client;
 
-        private ElasticClientSingleton()
+        private ElasticClientSingleton(IDictionary config = null)

```

Comment:
@jayhilden You're right, this is a private constructor which is always called with the parameter, it should not be optional